src/run-script.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <AssemblyName>r</AssemblyName>
```

Comment:
Should probably happen in its own PR. I don't think this is a breaking change since it's a tool and the command name didn't change, but might be best left for a v1 release.

Reply:
This change broke the integration job because the package name changed and was rolled back for a future update.